rotkehlchen/externalapis/etherscan.py

```diff
@@ -532,8 +534,8 @@ def get_token_transaction_hashes(
         yield _hashes_tuple_to_list(hashes)
 
     def has_activity(self, account: ChecksumEvmAddress) -> EtherscanHasChainActivity:
-        """Queries transactions, internal_txs and tokentx for an address with limit=1
-        just to quickly determine if the account has had any activity in the chain.
+        """Queries native asset balance, transactions, internal_txs and tokentx for an address
+        with limit=1 just to quickly determine if the account has had any activity in the chain.
```

Comment:
Not sure if this has a trailing whitespace as I am checking from mobile. If it does fix it in another PR

Reply:
I don't see any, ` make format` did the job I guess..